riscv/mmu.h

```diff
@@ -424,6 +460,9 @@ class mmu_t
   reg_t pmp_homogeneous(reg_t addr, reg_t len);
   reg_t pmp_ok(reg_t addr, reg_t len, access_type type, reg_t mode);
 
+#ifdef RISCV_ENABLE_DUAL_ENDIAN
+  bool target_big_endian;
+#endif
```

Comment:
bool target_big_endian;

Suggestion: let compiler optimize path by constant value and remove some ifdef in code

Reply:
You mean change this to
```
#ifdef RISCV_ENABLE_DUAL_ENDIAN
  bool target_big_endian;
#else
  const bool target_big_endian = false;
#endif
```
?  Sure, I can do that.